sim/src/test/scala/bridges/BridgeSuite.scala

```diff
@@ -0,0 +1,92 @@
+//See LICENSE for license details.
+
+package firesim.bridges
+
+import java.io.File
+
+import org.scalatest.Suites
+import org.scalatest.matchers.should._
+
+import freechips.rocketchip.config.Parameters
+import freechips.rocketchip.config._
+
+abstract class BridgeSuite(
+  val targetName:  String, // See GeneratorUtils
+  targetConfigs:   String      = "NoConfig",
+  platformConfigs: String      = "HostDebugFeatures_DefaultF1Config",
+  tracelen:        Int         = 8,
+  simulationArgs:  Seq[String] = Seq(),
+) extends firesim.TestSuiteCommon
+    with Matchers {
+
+  val backendSimulator = "verilator"
+
+  val targetTuple    = s"$targetName-$targetConfigs-$platformConfigs"
+  val commonMakeArgs = Seq(
+    s"TARGET_PROJECT=bridges",
+    s"DESIGN=$targetName",
+    s"TARGET_CONFIG=${targetConfigs}",
+    s"PLATFORM_CONFIG=${platformConfigs}",
+  )
+
+  def run(
+    backend:  String,
+    debug:    Boolean      = false,
+    logFile:  Option[File] = None,
+    waveform: Option[File] = None,
+    args:     Seq[String]  = Nil,
+  ) = {
```

Comment:
I won't let this block the PR, but you should always label return types moving forward.